src/renderer/store/components/App/index.js

```diff
@@ -41,17 +42,53 @@ const styleSheet = createStyleSheet('App', {
 const App = (props) => {
   const {
     classes,
+    sortBy,
+    sortOrder,
   } = props;
 
+  const renderTitleElement = () => {
+    let titleText;
+    if (sortBy) {
+      switch (sortBy) {
+        case 'installCount': {
+          titleText = sortOrder === 'asc' ? 'Least popular apps' : 'Most popular apps';
+          break;
+        }
+        case 'name': {
+          titleText = sortOrder === 'asc' ? 'Apps by name (A-Z)' : 'Apps by name (Z-A)';
+          break;
+        }
+        case 'createdAt': {
+          titleText = 'Most recently added apps';
```

Comment:
Should we use [title case](http://www.grammar-monster.com/lessons/capital_letters_title_case.htm) instead?